packages/client/components/RetroReflectPhase/PhaseItemEditor.tsx

```diff
@@ -46,12 +48,14 @@ const PhaseItemEditor = (props: Props) => {
     stackTopRef,
     cardsInFlightRef,
     setCardsInFlight,
-    dataCy
+    dataCy,
+    currentViewer
   } = props
   const atmosphere = useAtmosphere()
   const {onCompleted, onError, submitMutation} = useMutationProps()
   const [editorState, setEditorState] = useState(EditorState.createEmpty)
   const [isEditing, setIsEditing] = useState(false)
+  const [isAnonymous, setIsAnonymous] = useState(true)
```

Comment:
from a design perspective, why have a discrete state for each editor vs. a single `isAnonymous` state for the whole phase? no right or wrong, just curious for your thoughts